pages/Static.vue

```diff
@@ -1,167 +1,151 @@
-        },
-        {
-          title: i18n.t('Return policy'),
-          link: '/returns',
-          component: StaticExample
-        },
-        {
-          title: i18n.t('Privacy policy'),
-          link: '/privacy',
-          component: StaticShortExample
+          title: 'Privacy Policy',
+          message: `To help us achieve our goal of providing the highest quality products and
+      services, we use information from our interactions with you and other
+      customers, as well as from other parties. Because we respect your privacy,
+      we have implemented procedures to ensure that your personal information is
+      handled in a safe, secure, and responsible manner. We have posted this
+      privacy policy in order to explain our information collection practices
+      and the choices you have about the way information is collected and used.`
         },
         {
-          title: i18n.t('Size guide'),
-          link: '/size-guide',
-          component: StaticExample
+          title: 'Security',
+          message: `Personal information provided on the website and online credit card
+      transactions are transmitted through a secure server. We are committed to
+      handling your personal information with high standards of information
+      security. We take appropriate physical, electronic, and administrative
+      steps to maintain the security and accuracy of personally identifiable
+      information we collect, including limiting the number of people who have
+      physical access to our database servers, as well as employing electronic
+      security systems and password protections that guard against unauthorized
+      access.`
         },
         {
-          title: i18n.t('Contact us'),
-          link: '/contact',
-          component: StaticShortExample
+          title: 'Additional Information',
+          message: `This website ("website") is operated by Luma Inc., which includes Luma
+      stores, and Luma Private Sales. This privacy policy only covers
+      information collected at this website, and does not cover any information
+      collected offline by Luma. All Luma websites are covered by this privacy
+      policy.`
         }
       ]
     };
   },
   computed: {
-    activeComponent () {
-      const matchedNav = this.navigation.find(
-        nav => nav.link === this.$route.path
-      );
-      return matchedNav ? matchedNav.component : null;
+    breadcrumbs () {
+      return [
+        { text: i18n.t('Homepage'), route: { link: '/' } },
+        { text: this.currentRoute.title }
+      ];
+    },
+    currentRoute () {
+      return {
+        ...this.navigation.find(navigation => navigation.link === this.$route.path)
+      };
+    },
+    content () {
+      return this.currentRoute.isCms
+        ? {
+          title: this.$store.state.cmsPage.current.title,
+          message: this.$store.state.cmsPage.current.content
+        }
+        : this.messages;
```

Comment:
`this.messages` is array and above we have object `{title, message}` :thinking: Is it ok?

Reply:
This is supported in `AStatic` component in `content` prop, which expects either array or object. If `content` is an array of objects then `SfTabs` is used (each object will have own tab). Otherwise just plain header and message is shown in `AStatic`.

Reply:
ok, I get it. AStatic can handle one content or mutliple :ok_hand: